.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+# Python CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
+      - image: circleci/python:3.6.1
+
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # - image: circleci/postgres:9.4
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "test_requirements.txt" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
```

Comment:
@thraxil what does the training dash mean?

Reply:
Not 100% sure. Everything but the `command` step is just the boilerplate yaml that circleCI gives you when you set up a new Python app.